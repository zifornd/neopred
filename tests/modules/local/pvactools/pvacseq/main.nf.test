nextflow_process {

    name "Test Process PVACTOOLS_PVACSEQ"
    script "modules/local/pvactools/pvacseq/main.nf"
    process "PVACTOOLS_PVACSEQ"

    test("Should run without failures") {

        when {
            params {
                // define parameters here. Example:
                // outdir = "tests/results"
            }
            process {
                """
                // define inputs of the process here. Example:
		input[0] = Channel.of([
                    [ id:'SRR8281226' ], 
                    [ file("$projectDir/tests/test_result/vcfexpressionannotator/SRR8281226.mutect2.somatic.base.snp.Somatic.hc.filter.vep.gx.vcf", checkIfExists: true) ]
                ])
		input[1] = Channel.of([
                    [ id:'SRR8281226' ], 
                    [ file("$projectDir/tests/test_result/gunzip/SRR8281226.annotated.vcf", checkIfExists: true) ]
                ])

	        input[2] = "MHCflurry"
	        input[3] = "9"

                """
            }
        }

        then {
            assert process.success
	    assert process.out.versions
            assert snapshot(process.out).match()
        }

    }

}
