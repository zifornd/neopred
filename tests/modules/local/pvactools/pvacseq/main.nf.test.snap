{
    "Should run without failures": {
        "content": [
            {
                "0": [
                    
                ],
                "1": [
                    "versions.yml:md5,e7f8bdea05bd9bc57654ae0b7a1c4d22"
                ],
                "results": [
                    
                ],
                "versions": [
                    "versions.yml:md5,e7f8bdea05bd9bc57654ae0b7a1c4d22"
                ]
            }
        ],
        "meta": {
            "nf-test": "0.9.2",
            "nextflow": "24.10.1"
        },
        "timestamp": "2025-01-06T09:06:17.584119"
    }
}